src/PackageManagement.VisualStudio/Strings.resx

```diff
@@ -183,4 +183,10 @@
   <data name="Warning_ErrorDuringProjectClosureWalk" xml:space="preserve">
     <value>Failed to resolve all project references for '{0}'. The package restore result for '{1}' may be incomplete.</value>
   </data>
+  <data name="SolutionIsNotAvailable" xml:space="preserve">
+    <value>Solution is not available. Since, there is a packages.config based project and/or globalPackagesFolder setting is a relative path, NuGet needs a saved solution to determine the solution directory. Please save your solution before managing NuGet packages.</value>
+  </data>
+  <data name="RelativeGlobalPackagesFolder" xml:space="preserve">
+    <value>GlobalPackagesFolder is a relative path and solution is not saved. Please save your solution or configure a globalPackagesFolder path which is a full path</value>
```

Comment:
Updated string to be the following:
'globalPackagesFolder' from nuget.config file or the environment variable is '{0}', a relative path and the solution is not saved. Please save your solution or configure a 'globalPackagesFolder' which is a full path.